src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="block">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+    rel="preconnect" 
+    href="https://fonts.gstatic.com" 
+    crossorigin="anonymous"
+    >
+    <link 
+    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" 
+    rel="stylesheet"
+    >
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__img">
+        <img src="images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock" 
+          class="img"
+        >
+      </div>
+
+      <div class="card__information">
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+        <div class="card__code-num">Product code: 195434</div>
```

Comment:
Do not use div for text content, use semantic tags instead
Like h2 for heading and p for other kind of text